validation/standard/standard-prestates.toml

```
@@ -1,5 +1,13 @@
 latest_stable = "1.6.1"
-latest_rc = "1.6.1-rc.1"
+latest_rc = "1.7.0-rc.1"
+
+[[prestates."1.7.0-rc.1]]
```

```suggestion
[[prestates."1.7.0-rc.1"]]
```
Missing the closing quote here and below.